README.md

```diff
@@ -20,33 +20,51 @@ Consider using [fluo-dev] to run these requirements
 
 ### Configure your environment
 
-First, you must create the configuration files `data.yml` and `dropwizard.yml` files in the `conf/` 
-directory and edit them for your environment:
+First, you must create the configuration file `data.yml` in the `conf/` directory and edit it
+for your environment:
 
     cp conf/data.yml.example conf/data.yml
-    cp conf/dropwizard.yml.example conf/dropwizard.yml
+
+There are a few environment variables that need to be set to run these scripts (see `conf/webindex-env.sh.example`
+for a list).  If you don't want to set them in your `~/.bashrc`, create `webindex-env.sh` in `conf/` and set them.
+
+    cp conf/webindex-env.sh.example conf/webindex-env.sh
 
 ### Copy CommonCrawl data from AWS into HDFS
 
 CommonCrawl data sets are hosted in S3.  The following command will run a Spark job that copies
-data files from AWS into HDFS.  This script is configured by `conf/data.yml` where you can specify
-the HDFS data directory, number of files to copy, and number of executors to run.
+data files from AWS into HDFS `init/` and `load/` directoires.  This script is configured by 
+`conf/data.yml` where you can specify the number of files to copy into each directory:
 
-    ./bin/copy.sh
+    ./bin/webindex copy
 
 ### Initialize Fluo & Accumulo with data
 
-After you have copied data to HDFS, run the following command to run a Spark job that will index
-your data in HDFS and initialize Accumulo and Fluo with data.
+After you have copied data into HDFS, run the following command to run a Spark job that will initialize 
+Fluo and Accumulo with data in your HDFS `init/` directory.
+
+    ./bin/webindex init
 
-    ./bin/init.sh
+### Load files into Fluo
 
-### Run the web application
+The `init` command can only be run on an empty cluster.  To add more data, run the `load` which starts
+a Spark job that loads Fluo with data in your HDFS `load/` directory.  Fluo will incrementally process 
+this data and update Accumulo.
 
-Finally, run the following command to run the web app:
+    ./bin/webindex copy
 
-    ./bin/webapp.sh
+### Run the webindex UI
+
+Run the following command to run the webindex UI:
+
+    ./bin/webindex ui
```

Comment:
I wasn't sure about it going to a log because the webserver will just run forever.  However, I like output going to a log too.  I am sure users will know to kill it when finished.  Fixed in 5fbb5941992c95